bundle/config/workspace.go

```diff
@@ -101,6 +101,19 @@ func (w *Workspace) Client() (*databricks.WorkspaceClient, error) {
 		AzureLoginAppID:  w.AzureLoginAppID,
 	}
 
+	for k := range config.ConfigAttributes {
+		attr := &config.ConfigAttributes[k]
+		if !attr.IsZero(cfg) {
+			attr.SetSource(&config.Source{Type: config.SourceType("bundle")})
+		}
```

Comment:
Mentioned this in the SDK PR as well; this is a global. We should seek a solution that stores these details on the configuration object instead. It could still use properties from `attr` to do so, e.g. the field index.